CONTRIBUTING.md

```diff
@@ -244,6 +244,11 @@ We prefer to import core traits, e.g. using `TryFrom` instead of
 
 ### Misc
 
+#### Numeric literals
+
+We generally prefer numeric literals to be written in hexadecimal instead of
```

Comment:
I would qualify this more? Consider picking a base to fit in the with the domain in which the value is used. For example, you might want to use hexadecimal for protocol message encoding and octal for Unix privileges.

Maybe also mention something about the digit grouping here, as in `1_000_000`?

Reply:
> I would qualify this more? Consider picking a base to fit in the with the domain in which the value is used.

Borrowed your wording for this :+1: 

> Maybe also mention something about the digit grouping here, as in 1_000_000?

Done.